modules/caddyhttp/rewrite/rewrite.go

```diff
@@ -132,6 +132,11 @@ func (rewr *Rewrite) Provision(ctx caddy.Context) error {
 func (rewr Rewrite) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyhttp.Handler) error {
 	repl := r.Context().Value(caddy.ReplacerCtxKey).(*caddy.Replacer)
 
+	if rewr.logger.Check(zap.DebugLevel, "rewrote request") == nil {
```

Comment:
```suggestion
	if rewr.logger.Check(zap.DebugLevel, "rewrote request") == nil {
```

"rewrote request" should be extracted to a const

Reply:
done 👍